client/my-sites/plan-features-2023-grid/_media-queries.scss

```diff
@@ -27,6 +39,17 @@ $minWidthToGridWidthMap: (
 
 }
 
+@mixin plan-comparison-grid-breakpoints {
+	width: 100%;
+
+	@each $screenMinWidth, $gridWidth in $planComparisonMinWidthToGridWidthMap {
+		@media ( min-width: #{$screenMinWidth} ) {
+			width: #{$gridWidth};
+		}
+	}
+
```

Comment:
Nit: a redundant newline here :)

Reply:
Removed in [dbc5a28](https://github.com/Automattic/wp-calypso/pull/77039/commits/dbc5a2857ca581e100a77ddfd3ef19d807407259)